test/quantization/pt2e/test_metadata_porting.py

```diff
@@ -58,13 +58,43 @@ def _tag_partitions(
 
 
 class TestMetaDataPorting(QuantizationTestCase):
+    def _test_quant_tag_preservation_through_decomp(
+        self, model, example_inputs, from_node_to_tags
+    ):
+        ep = export.export(model, example_inputs)
+        found_tags = True
+        not_found_nodes = ""
+        for from_node, tag in from_node_to_tags.items():
+            for n in ep.graph_module.graph.nodes:
+                from_node_meta = n.meta.get("from_node", None)
+                if from_node_meta is None:
+                    continue
+                if not isinstance(from_node_meta, list):
+                    raise ValueError(
+                        f"from_node metadata is of type {type(from_node_meta)}, but expected list"
+                    )
```

Comment:
nit: is "from_node" a known metadata? can we just assume this is true?

also can you add a comment with an example "from_node"

Reply:
> nit: is "from_node" a known metadata? can we just assume this is true?

It is what is just being tested.